Package.swift

```diff
@@ -31,7 +31,7 @@ let package = Package(
   dependencies: [
     .package(
         url: "https://github.com/google/abseil-cpp-binary.git",
-        "1.2021110200.0" ..< "1.2021110300.0"
+        "1.2022062300.0" ..< "1.2022062400.0"
```

Comment:
Corresponding grpc-ios repo's abseil pinning for 1.50.1 release: https://github.com/grpc/grpc-ios/blob/6568b62c608c2776da9e8d0842dea4c43993095c/Package.swift#L23